public/app/plugins/datasource/graphite/FunctionEditor.tsx

```diff
@@ -68,6 +68,16 @@ class FunctionEditor extends React.PureComponent<FunctionEditorProps, FunctionEd
                   onClick={popperProps.show ? hidePopper : showPopper}
                   style={{ cursor: 'pointer' }}
                 >
+                  {this.props.func.def.unknown && (
+                    <Tooltip content={<TooltipContent />} placement="bottom">
+                      <Icon
+                        data-testid="warning-icon"
+                        name="exclamation-triangle"
+                        size="xs"
+                        style={{ marginRight: '2px' }}
```

Comment:
NIT: you might want to use `spacing.xss` from grafana theme

something like:
```
const theme = useTheme();
// ..
<Icon
  className={css`margin-right: ${theme.spacing.xss}`}
/>

Reply:
i also wonder if we could use `aria-role="alert"` to make it more a11y-friendly, but I don't know how nicely it plays with the tooltip

Reply:
> NIT: you might want to use spacing.xss from grafana theme

I like it! And sorry but I ended up making the component functional :P 

> i also wonder if we could use aria-role="alert" to make it more a11y-friendly, but I don't know how nicely it plays with the tooltip

I was thinking about it but according to [docs](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_alert_role) it might be too verbose:

> (...) Because of its intrusive nature, the alert role must be used sparingly and only in situations where the user's immediate attention is required.  (...)